core/src/main/scala/io/projectglow/vcf/VariantContextToInternalRowConverter.scala

```diff
@@ -285,6 +286,48 @@ class VariantContextToInternalRowConverter(
     makeArray(strList, parseFn)
   }
 
+  private def getAnnotationArray(
```

Comment:
Can we write this in Java or use performant scala style? Whenever we use scala collections like this we end up rewriting for performance.